include/openmc/particle_data.h

```diff
@@ -309,7 +309,9 @@ class ParticleData {
   int n_event_ {0}; // number of events executed in this particle's history
 
   // Weight window information
-  int n_split_ {0}; // Number of splits this particle has undergone
+  int n_split_ {0}; // Number of times this particle has been split
+  // Particle-specific factor for on-the-fly weight window adjustment
+  double ww_factor_ {0.0};
```

Comment:
The comments here look a little weird (like the first one for `n_split_` runs into the one above `ww_factor_`). Not sure if this is the fault of clang-format or what

Reply:
Hrmm yeah I'm not sure. I'll take a look.